conans/test/functional/utils.py

```diff
@@ -34,3 +34,29 @@ def check_msc_ver(toolset, output):
         assert 10 <= version < 20, "Error:{}".format(output)
     else:
         raise NotImplementedError()
+
+
+def check_msvc_library(library, client, vs_version, build_type, static, architecture="amd64"):
```

Comment:
This really need to be unified with the existing ``check_vs_runtime()``

Reply:
Done.